extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -7,7 +7,11 @@
  * @author Krzysztof Krzyżaniak <eloy@wikia-inc.com>
  */
 
-ini_set( "include_path", dirname(__FILE__)."/../../../../maintenance/" );
+use Swagger\Client\Discussion\Api\SitesApi;
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Swagger\ApiProvider;
```

Comment:
They should but you probably want to move the use statements below this `ini_set` and the two `require_once` statements, as those setup the Wikia environment.